www/runtest.php

```diff
@@ -2170,6 +2188,10 @@ function CreateTest(&$test, $url, $batch = 0, $batch_locations = 0)
                 AddIniLine($testFile, 'lighthouse', '1');
             if( isset($test['lighthouseTrace']) && $test['lighthouseTrace'] )
                 AddIniLine($testFile, 'lighthouseTrace', '1');
+            if( $test['heroElementTimes'] )
+                AddIniLine($testFile, 'heroElementTimes', '1');
+            if( $test['heroElements'] && strlen($test['heroElements']) )

```

Comment:
can you change the first part of this check to isset($test['heroElements']) ?

Reply:
Thanks - done